interactive/printer.go

```diff
@@ -18,7 +19,10 @@ func QuoteMarkedFiles(markedFiles map[*core.File]struct{}) []string {
 	quotedFiles := make([]string, len(markedFiles))
 	i := 0
 	for file := range markedFiles {
-		quotedFiles[i] = fmt.Sprintf("'%s'", file.Path())
+		p := file.Path()
+		// Escape single quotes
+		p = strings.Replace(p, "'", "\\'", -1)
```

Comment:
Great change 👍, would you mind writing a small test for this logic?

Reply:
Added & squashed